courses/fundamentals_of_ada/030_basic_types.rst

```diff
@@ -1429,34 +1038,6 @@ Assignment Respects Constraints
    J : Integer  := P; -- always legal
    K : Integer  := N; -- always legal
 
-----------------------------------------
```

Comment:
This is important to beginners, otherwise they'll never know why their subtypes behave "wrong"